content/blog/last-call-sasl-external.md

```diff
@@ -7,7 +7,7 @@ categories: ["Miscellaneous"]
 
 Today we [issued](https://mail.jabber.org/pipermail/standards/2007-January/013676.html) a Last Call regarding [XEP-0178: Best Practices for Use of SASL EXTERNAL](https://xmpp.org/extensions/xep-0178.html). This specification is part of our [continuing effort](http://blog.xmpp.org/?p=4) to advance the XMPP RFCs to Draft Standard within the IETF.
```

Comment:
[`?p=4`](https://web.archive.org/web/20070629021227/http://blog.xmpp.org/?p=4) looks like https://xmpp.org/2007/01/continuing-the-conversation-at-the-ietf/